app/main.py

```diff
@@ -1,3 +1,80 @@
+import math
+
+
 class Vector:
-    # write your code here
-    pass
+    def __init__(self, x, y):
+        self.x = round(x, 2)
+        self.y = round(y, 2)
+
+    def __add__(self, other):
+        if type(other) == int or type(other) == float:
```

Comment:
It doesn't look obvious to me. I'm not sure that class `Vector` should support addition with a number, it could lead to unexpected behavior instead of an error.